components/ai-story-generator-backup.tsx

```
@@ -0,0 +1,919 @@
+                  </Button>
+                </motion.div>
+              </div>
+              <motion.div 
+                className="bg-gradient-to-r from-primary/10 via-amber-500/5 to-primary/10 p-1 rounded-lg overflow-hidden relative"
+                initial={{ scale: 0.98, opacity: 0.8 }}
+                animate={{ scale: 1, opacity: 1 }}
+                transition={{ delay: 0.4, duration: 0.6, ease: "easeOut" }}
+              >
+                <div className="absolute inset-0 bg-gradient-to-br from-transparent via-white/10 to-transparent opacity-30"></div>
+                <div className="prose prose-lg dark:prose-invert max-h-96 overflow-y-auto p-6 bg-background rounded-md relative z-10">
+                  {generatedContent ? <div dangerouslySetInnerHTML={{ __html: generatedContent }} /> : <p>No content generated yet.</p>}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)